### PR TITLE
Use Cockpit's Kebab dropdown

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

File: src/Containers.jsx [hunk 1]
```diff
@@ -243,7 +222,7 @@ const ContainerActions = ({ container, healthcheck, onAddNotification, localImag
         }
         if (container.isSystem && container.State?.CheckpointPath) {
             actions.push(
-                <DropdownSeparator key="separator-0" />,
+                <Divider key="separator-0" />,
```
Comment (cockpituous) [inline src/Containers.jsx:225]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1605-20240306-082630-f936f937-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: test/check-application [hunk 2]
```diff
@@ -916,7 +916,7 @@ class TestApplication(testlib.MachineCase):
 
         # Delete intermediate image which is in use
         self.execute(auth, f"podman untag {IMG_INTERMEDIATE}")
-        b.click(intermediate_image_sel + " .pf-v5-c-dropdown__toggle")
+        b.click(intermediate_image_sel + " .pf-v5-c-menu-toggle")
```
Comment (jelly) [inline test/check-application:919]: Maybe we should just use a .... helper function

Comment (jelly) [reply test/check-application]: This becomes more and more of a yakshave as we repeat the same stuff in `deleteImage`... This feels follow up territory.

**Review by martinpitt — APPROVED** [review]
Niiice! and no pixel changes any more! ^5s

**Pull request merged**